crates/bevy_sprite/src/picking_backend.rs

```diff
@@ -11,25 +11,50 @@ use bevy_ecs::prelude::*;
 use bevy_image::Image;
 use bevy_math::{prelude::*, FloatExt, FloatOrd};
 use bevy_picking::backend::prelude::*;
+use bevy_reflect::prelude::*;
 use bevy_render::prelude::*;
 use bevy_transform::prelude::*;
 use bevy_window::PrimaryWindow;
 
+/// Runtime settings for the [`SpriteBackend`].
+#[derive(Resource, Reflect)]
+#[reflect(Resource, Default)]
+pub struct SpriteBackendSettings {
+    /// When set to `true` picking will ignore any part of a sprite which is transparent
+    /// Off by default for backwards compatibility. This setting is provided to give you fine-grained
+    /// control over if transparency on sprites is ignored.
+    pub passthrough_transparency: bool,
+    /// How Opaque does part of a sprite need to be in order count as none-transparent (defaults to 10)
+    pub transparency_cutoff: u8,
+}
+
+impl Default for SpriteBackendSettings {
+    fn default() -> Self {
+        Self {
+            passthrough_transparency: false,
```

Comment:
The mix of transparency as a prefix and suffix feels like this should have a different name, but I don’t have any good suggestions right now. 

Also, worth checking if transparency is the right term compared to alpha. No strong opinions but it stood out to me as a question.

Reply:
I think you're right. Alpha is absolutely a better term. Updating.